docker-compose.yaml

```diff
@@ -218,90 +218,98 @@ services:
     ports:
       - "3000:3000"
 
-#  etcd:
```

Comment:
Do we really want to default these to deploy?

Reply:
note the added
profiles:
  - retrieval

they will only deploy if you pass the `--profile retrieval` arg to docker compose up

Reply:
They won't deploy by default. The user must pass in the `--profile retrieval` argument for them to deploy. Like

`docker compose up --profile retrieval` if they leave that off they will not be deployed.